### DISCUSSION TITLE
Sharing Library panel as part of sharing the dashboard

### DISCUSSION THREAD

**Discussion by gaurkuma** [grafana/grafana#38097] [Dashboards, Panels & Visualizations]
I have a dashboard which makes use of panels picked up from a Panel Library. But when i share the dashboard on to another system, it errors out as it is not able to find these panels. What's the best way to copy the panel library as i am not sure where they reside. Also, is there a better one-click way of sharing dashboards which used library panels?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
